stopwatch/stopwatch.go

```diff
@@ -0,0 +1,63 @@
+// Package stopwatch provides a simple stopwatch component.
+package stopwatch
+
+import (
+	"time"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// TickMsg is a message that is sent on every timer tick.
+type TickMsg struct{}
+
+// StopMsg is a message that can be send to stop the watch.
+type StopMsg struct{}
+
+// Model of the timer component.
+type Model struct {
```

Comment:
Should we add a method for returning the time elapsed since the stopwatch started?

Reply:
done !